core/bin/zksync_server/src/node_builder.rs

```diff
@@ -122,7 +122,6 @@ impl MainNodeBuilder {
         let pools_layer = PoolsLayerBuilder::empty(config, secrets)
             .with_master(true)
             .with_replica(true)
-            .with_prover(true) // Used by house keeper.
```

Comment:
That's great! Is it now possible to remove `prover_dal` from the [pools_layer](https://github.com/matter-labs/zksync-era/blob/be76766e76f1a4321eaace3fc33ea2f409fe8837/core/node/node_framework/src/implementations/layers/pools_layer.rs) and move it from the [core workspace](https://github.com/matter-labs/zksync-era/blob/be76766e76f1a4321eaace3fc33ea2f409fe8837/Cargo.toml#L82-L83) to the prover workspace?

So far it was the most confusing part regarding workspace composition.